src/main/java/pages/JacketsMenPage.java

```diff
@@ -0,0 +1,52 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class JacketsMenPage {
+
+    protected WebDriver driver;
+    protected Actions action;
+    private By cardOfJackets = By.className("product-item-info");
+    private By sizeButtonS = By.id("option-label-size-143-item-167");
+    private By colorButton = By.id("option-label-color-93-item-56");
+    private By addToCartButton = By.className("action tocart primary");
+    private By succesMessage = By.className("message-success success message");
+    private By iconCart = By.className("action showcart");
+    private By  checkoutButton = By.id("top-cart-btn-checkout");
+
+    public JacketsMenPage (WebDriver driver,Actions action) {
+        this.driver = driver;
+        this.action = action;
+    }
+
+    public void hoverCardOfJacket(){
+        WebElement cardOfJackets = driver.findElement(By.className("product-item-info"));
+    }
+    public void  clickSizeButtonS(){
```

Comment:
the last letter is not necessary in the method name

Reply:
Done